com.tracktion.Waveform.yml

```diff
@@ -0,0 +1,138 @@
+id: com.tracktion.Waveform
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: waveform
+finish-args:
+  - --device=all
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2
+  - --env=LXVST_PATH=/app/extensions/Plugins/lxvst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --filesystem=home/.lxvst:create
+  - --filesystem=home/.vst:create
+  - --filesystem=home/.vst3:create
+  - --filesystem=home/Tracktion
+  - --filesystem=xdg-config
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-videos
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /cache
+  - /include
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '20.08'
+    add-ld-path: lib
+    merge-dirs: ladspa;lv2;lxvst;vst3
```

Comment:
still does it support LADSPA and LV2? if not then this should be shorten and the env should be reduced too.

Reply:
Waveform11 supports LADSPA, but not LV2 apparently.